src/foundation.cs

```diff
@@ -8827,15 +9775,44 @@ interface NSItemProviderReading
 		//[Export ("readableTypeIdentifiersForItemProvider", ArgumentSemantic.Copy)]
 		//string[] ReadableTypeIdentifiersForItemProvider { get; }
 
+		//[Static]
+		//[Export ("objectWithItemProviderData:typeIdentifier:error:")]
+		//[return: NullAllowed]
+		//INSItemProviderReading GetObject (NSData data, string typeIdentifier, [NullAllowed] out NSError outError);
+	}
+
+	interface INSItemProviderWriting {}
+
+	[Watch (4,0), TV (11,0), Mac (10,13), iOS (11,0)]
+	[Protocol, Model]
+	[BaseType (typeof(NSObject))]
+	interface NSItemProviderWriting
+	{
 		//
-		// This is a constructor that various classes must implement
-		// NSAttributedString, UIColor and UIImage need to have this constructor
-		// for user-defined implementations of this interface, we are going to
-		// need to something special
+		// This static method has to be implemented on each class that implements
+		// this, this is not a capability that exists in C#.
+		// We are inlining these on each class that implements NSItemProviderWriting
+		// for the sake of the method being callable from C#, for user code, the
+		// user needs to manually [Export] the selector on a static method, like
+		// they do for the "layer" property on CALayer subclasses.
 		//
-		//[Abstract]
-		//[Export ("initWithItemProviderData:typeIdentifier:error:")]
-		//INSItemProviderReading CreateFrom (NSData providerData, string typeIdentifier, [NullAllowed] NSError outError);
+		//[Static, Abstract]
+		//[Export ("writableTypeIdentifiersForItemProvider", ArgumentSemantic.Copy)]
+		//string[] WritableTypeIdentifiersForItemProvider { get; }
+
+		//[Static]
+		//[Export ("itemProviderVisibilityForRepresentationWithTypeIdentifier:")]
+		//NSItemProviderRepresentationVisibility GetItemProviderVisibility (string typeIdentifier);
+
+		[Export ("writableTypeIdentifiersForItemProvider", ArgumentSemantic.Copy)]
+		string[] WritableTypeIdentifiersForItemProvider { get; }
```

Comment:
That's `[Static]` elsewhere, including the comment just above ?!? why the instance version ?